commercetools-models/src/main/java/io/sphere/sdk/carts/ItemShippingDetails.java

```diff
@@ -23,7 +23,7 @@
      * Convenience method to extract map address keys to their target without further processing
      * @return Map with addressKey as key and destined quantity as value.
      */
-    default Map<String, Integer> getTargetsMap() {
+    default Map<String, Long> getTargetsMap() {
```

Comment:
We should document this breaking change in our release notes.

Reply:
good catch 👍